input/docs/handbook/commands/index.md

```diff
@@ -1,23 +1,175 @@
+`ReactiveCommand` is a Reactive Extensions and asynchronous aware implementation of the [`ICommand`](https://msdn.microsoft.com/en-us/library/system.windows.input.icommand.aspx) interface. `ICommand` is often used in the [MVVM design pattern](https://docs.microsoft.com/en-us/dotnet/framework/wpf/advanced/commanding-overview) to allow the View to trigger business logic defined in the ViewModel. This allows for easier maintenance, unit testing, and the ability to reuse ViewModels across different UI frameworks. Examples of where a View might invoke a command include clicking a *Save* menu item, tapping a phone icon, or stretching an image. In these cases, the ViewModel will then invoke the business logic of saving outstanding changes, performing a phone call, or zooming into an image.
 
 `ReactiveCommand` adds the concept of Input and Output generic types. The Input is often passed in by the View, and the Output is the result of executing the command. ReactiveCommand's are `IObservable<TOutput>` which can be used like any other IObservable. For example, since the `ReactiveCommand` is `IObservable<TOutput>` you can `Subscribe()` to it like any other observable, and add the output to a List on your view model. The `Unit` type is a functional programming construct analogous to void and can be used in cases where you don't care about either the input and/or output value.
 
-`ReactiveCommand` has full support for asynchronous logic and will block re-execution while executing. `ReactiveCommand` guarantees the result of events are delivered to the provided `IScheduler` (which defaults to the main thread scheduler). The executing logic thread safety is the user's responsibility but any result from the logic is guaranteed to arrive on the specified `IScheduler`.
+# Creating commands
 
 A `ReactiveCommand` is created using static factory methods which allows you to create command logic that executes either synchronously or asynchronously. The following are the different static factory methods:
 
 * `CreateFromTask()` - Executes a C# [Task Parallel Library (TPL)](https://docs.microsoft.com/en-us/dotnet/standard/parallel-programming/task-based-asynchronous-programming) Task. This allows use also of the C# [async/await](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/async) operators.
 * `CreateFromObservable()` - Execute the logic using an `IObservable`.
```

Comment:
CreateFromObservable should probably be the top item. It's what we usually recommend over CreateFromTask